src/ngraph/frontend/fluid/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ target_sources (ngraph PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}/operators/pool.hpp
     ${CMAKE_CURRENT_SOURCE_DIR}/operators/reduce_sum.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/operators/reduce_sum.hpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/operators/matmul_pd.hpp
```

Comment:
You may move those up above line 19 to make them in alphabetic order. You can rename the file to matmul instead of matmul_pd since we are in a different namespace.